website/app/templates/index.hbs

```diff
@@ -86,14 +86,14 @@
         @color='var(--brand)'
         class='sm:mr-0 md:mr-2 lg:mr-4'
       />
-      GitHub Repository
+      GitHub repository
     </a>
   </div>
 
 </div>
 
 <section class='ds-section' aria-labelledby='available-icons'>
-  <h2 class='ds-h2 mb-3' id='available-icons'>Available Icons</h2>
+  <h2 class='ds-h2 mb-3' id='available-icons'>Available icons</h2>
```

Comment:
Note: Prettier is not working here, but I've extracted that to a separate issue: https://github.com/hashicorp/flight/issues/365